### ISSUE TITLE
Logging "done"

### ISSUE THREAD

**Issue by 2ndalpha** [marceloverdijk/lesscss-java#59]
After compilation it logs "done" to the console. It's annoying and not needed.
See: https://github.com/marceloverdijk/lesscss-java/blob/a7102a49d891b034b3dbcc6e5ccd74fbb3d2fa05/src/main/resources/META-INF/lessc-rhino-1.7.0.js#L449

Could it be removed?

**Comment by nalroff** [comment]
I agree. Or if there's a way to disable the output, it should be documented.
